spec/erb_lint/cli_spec.rb

```diff
@@ -171,7 +171,13 @@ def run(_processed_source)
             FileUtils.mkdir_p(linted_dir)
           end
 
-          it { expect { subject }.to(output(/no files found/).to_stdout) }
+          it 'fails' do
+            expect(subject).to(be(false))
+          end
+
+          it 'shows errors were found to stderr' do
```

Comment:
Since this same test case name is used for something that actually outputs "no errors were found" (line 145), it may be good to be more specific. Something like `shows no files were found to stderr`.